--- utils/mount/network.c.org	Fri Jan 13 00:21:39 2017
+++ utils/mount/network.c	Wed Jan 25 11:34:18 2017
@@ -38,7 +38,6 @@
 #include <sys/socket.h>
 #include <sys/wait.h>
 #include <sys/stat.h>
-#include <linux/in6.h>
 #include <netinet/in.h>
 #include <rpc/rpc.h>
 #include <rpc/pmap_prot.h>
@@ -1125,6 +1124,7 @@
 		if (bind(sock, SAFE_SOCKADDR(&sin), sizeof(sin)) < 0)
 			goto out;
 		break;
+#if 0
 	case AF_INET6:
 		/* Make sure the call-back address is public/permanent */
 		val = IPV6_PREFER_SRC_PUBLIC;
@@ -1132,6 +1132,7 @@
 		if (bind(sock, SAFE_SOCKADDR(&sin6), sizeof(sin6)) < 0)
 			goto out;
 		break;
+#endif
 	default:
 		errno = EAFNOSUPPORT;
 		goto out;


